core/src/test/java/com/predic8/membrane/core/util/URLParamUtilTest.java

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2023 predic8 GmbH, www.predic8.com
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. */
+
+package com.predic8.membrane.core.util;
+
+import com.predic8.membrane.core.exchange.*;
+import com.predic8.membrane.core.http.*;
+import org.junit.jupiter.api.*;
+
+import java.io.*;
+import java.net.*;
+
+import static com.predic8.membrane.core.http.Header.*;
+import static com.predic8.membrane.core.http.MimeType.*;
+import static com.predic8.membrane.core.util.URLParamUtil.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class URLParamUtilTest {
```

Comment:
please also add this test to the `UnitTests` suite

Reply:
I checked out the branch and the Test is included in the UnitTest? I also pushed.